File: java/UberBlack.java
import java.util.ArrayList;
import java.util.Map;

class UberBlack extends Car {
    Map<String, ArrayList<String, Integer>> TypeCarAccepted;
    ArrayList<String> seatsMaterial;

    public UberBlack(String license, Account driver,
    Map<String, ArrayList<String, Integer>> TypeCarAccepted,
    ArrayList<String> seatsMaterial) {
        super(license, driver);
        this.TypeCarAccepted = TypeCarAccepted;
        this.seatsMaterial = seatsMaterial;
    }
}

File: php/uberBlack.php
<?php
    require_once('car.php');
    class UberBlack extends Car {
        public $typeCarAccepted;     
        public $material;  
        
        public function __construct($license, $driver, $typeCarAccepted, $material) {
            parent::__construct($license,$driver);
            $this->typeCarAccepted = $typeCarAccepted;
            $this->material = $material;
        }
    }
?>